projects/aca-folder-rules/src/lib/rule-details/actions/rule-action.ui-component.html

```diff
@@ -16,6 +16,7 @@
   <adf-card-view
     data-automation-id="rule-action-card-view"
     [properties]="cardViewItems"
+    [ngStyle]="isFullWidth && {'width': '100%'}"
```

Comment:
should there be OR instead of AND ?

Reply:
> should there be OR instead of AND ?

I don't think so - when isFillWidth = true => set width to 100%.
Correct me if I am wrong

Reply:
If I'm not mistaken this should just evaluate to `true` if `isFullWidth` is true?

Reply:
Wait no I get it, it does evaluate to `{'width': '100%'}`. Can you create a getter for this?
```
get cardViewStyle {
  return this.isFullWidth ? {'width': '100%'} : {};
}
```

Reply:
done